packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md

```diff
@@ -1,37 +1,42 @@
-  * **Variable Name** &mdash; an automatically generated variable used when constructing the dynamic address. This is hidden from admins and is only passed to your external application.
-  * **Help Text** &mdash; any descriptive text to be shown to administrators when installing your app integration.
-  * **Type** &mdash; the property type for your parameter. Options are "String", "URL", or "HTTPS URL".
+* **Does your Redirect URI vary per tenant?**: If **Yes**, a new field appears to assist you in setting up a per tenant configuration.
+  * **What variables do your admins need to specify to install your app?**: When you click **Add Variable**, the interface displays a dialog box to collect the following information:
+    * **Label Name**: A descriptive name for the dynamic variable that administrators see when installing your app integration.
+    * **Variable Name**: An automatically generated variable used when constructing the dynamic address. This is hidden from admins and is only passed to your external application.
+    * **Help Text**: Any descriptive text to be shown to administrators when installing your app integration.
+    * **Type**: The property type for your parameter. Options are "String", "URL", or "HTTPS URL".
   * Click **Save** to add the variable to the list.
 
   After the variable is created, you can click the pencil icon to make changes to the details, the clipboard icon to copy the **Variable Name** to your local clipboard, or the "X" icon to remove the variable entirely.
 
-* **Construct your dynamic Redirect URI by copying the variables above and pasting them where applicable** &mdash; provide one or more complete sign-in redirect URIs where Okta sends the OAuth responses for your app integration. You must add at least one valid redirect URI.
+* **Construct your dynamic Redirect URI by copying the variables above and pasting them where applicable**: Provide one or more complete sign-in redirect URIs where Okta sends the OAuth responses for your app integration. You must add at least one valid redirect URI.
 
   If you're using a per tenant design, include the variable names that you created. For example:
   * https://`${app.variableName}`.okta.com
   * https://okta-`${app.variableName}`.com
+  * `${app.variableName}/route`
 
-* **Optional: Do you require an Initiate Login URI** &mdash; include a login URI if you want the Okta integration to handle redirecting your users to your application to start the sign-in request. When end users click a tile in their Okta dashboard, they are redirected to the `initiate_login_uri` of the client application, which constructs an authorization request and redirects the end user back to Okta.
+  > **Note**: A variable can include a complete URL (for example, https://example.com). This enables you to use more globally useful variables such as `${org.baseURL}`.
```

Comment:
@ScottRussell-okta : I moved the note below the variable examples and added $,{,} around org.baseURL. Please confirm if this edit is OK. Thanks

Reply:
This looks good to me.  Just to verify, the note section is added to Oidc/Saml/Scim tab's `submit.md` equally?

Reply:
Yes, the same note is in all 3 `submit.md` files.